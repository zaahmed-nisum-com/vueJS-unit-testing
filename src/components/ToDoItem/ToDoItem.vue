<template>
  <div></div>
</template>

<script>
export default {
  props: ["item"],
};
</script>

<style></style>
