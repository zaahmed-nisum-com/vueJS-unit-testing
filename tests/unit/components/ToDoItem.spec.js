import { createLocalVue, mount, shallowMount } from '@vue/test-utils'
import ToDoItem from '../../../src/components/ToDoItem/ToDoItem.vue'

describe('ToDoItem.vue', () => {
    test('snapshot', () => {
        const wrapper = mount(ToDoItem, {
            propsData: {
                item: []
            }
        })
        expect(wrapper.find('div')).toMatchSnapshot(true)
    })

})