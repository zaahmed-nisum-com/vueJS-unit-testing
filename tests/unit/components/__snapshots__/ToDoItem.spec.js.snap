// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home.vue snapshot 1`] = `<div></div>`;

exports[`ToDoItem.vue snapshot 1`] = `<div></div>`;
